app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1544,6 +1803,18 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
                 viewModel.onUserDismissedCta(cta)
             }
 
+            if (isExperimentEnabled()){
+                if (lastSeenOmnibarViewState != null){
+                    if (lastSeenOmnibarViewState!!.isEditing){
```

Comment:
We could get rid of `!!` using `let`

Reply:
`lastSeenOmnibarViewState` is mutable and smart cast won't work

Reply:
that's true, but using `let` you get a local immutable reference, and then you can do the smart casting with that one.

Reply:
I don't see a real benefit but I don't feel strongly about so I'll change it 👍